.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Need help? 
+Before opening a new issue, did you ask your question on
+* [Google group](http://groups.google.com/group/testng-users)
+* [StackOverflow](http://stackoverflow.com/questions/tagged/testng)
+
+If post on both sites, please provide the link to the other question to avoid duplicating the answer.
+
+# Are you sure it is a TestNG bug?
+Before posting the issue, try to reproduce the issue in [a shell window](http://testng.org/doc/documentation-main.html#running-testng).
+
+If the problem does not exist with the CLI, first check if the issue exists on the bugtracker of the runner, and open an issue there first:
+* Eclipse: https://github.com/cbeust/testng-eclipse/issues
+* IntelliJ: [https://youtrack.jetbrains.com/issues](https://youtrack.jetbrains.com/issues?q=%23%7BUnit%20Testing.%20TestNG%7D%20)
+* Maven: https://issues.apache.org/jira/browse/SUREFIRE
+* Gradle: https://issues.gradle.org/projects/GRADLE
+
+# Which version are you using?
+Always make sure your issue is happening on the latest TestNG version. Bug reports occurring on older versions will not be looked at quickly.
```

Comment:
Yep ;) I want to explain that issue on the latest version will have more priority.